config/config.go

```diff
@@ -90,6 +91,9 @@ type Config struct {
 
 	// WorkflowRestrictions restricts the controller to executing Workflows that meet certain restrictions
 	WorkflowRestrictions *WorkflowRestrictions `json:"workflowRestrictions,omitempty"`
+
+	//configurable initial delay https://github.com/argoproj/argo/issues/4107
```

Comment:
Might be good to say why you'd want to change this.

Reply:
edited